tests/models/test_onnx.py

```diff
@@ -77,6 +78,28 @@ def test_model_saves_with_example_input_array(tmpdir):
     assert os.path.getsize(file_path) > 4e2
 
 
+def test_model_with_two_inputs_saves(tmpdir):
```

Comment:
let's combine these tests with the above ones, and we can parametrize the inputs like so

```py
@pytest.mark.parametrize("input_sample", [torch.rand(..), (torch.rand(), torch.rand()) ]
def test_name(input_sample) 
```

Reply:
good idea, done